### PR TITLE
api - monitoring - exclude technical routes

### PR DESCRIPTION
**Pull request by brandicorel**

### CODE REVIEW

File: src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs [hunk 1]
```diff
@@ -53,14 +53,22 @@ public static ILogger CreateLogger(IConfiguration configuration, Action<LoggerCo
                                .ReadFrom.Configuration(configuration)
                                .Enrich.FromLogContext()
                                .Enrich.WithProperty("server-host", host)
-                               .Filter.With(new ExcludedPathFilter("/swagger", "/health", "/favicon.ico"))
+                               .Filter.With(
+                                    new ExcludedPathFilter(
+                                        "/swagger",
+                                        "/health",
```
Comment (brandicorel) [inline src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs:59]: should I keep this one?

File: src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs [hunk 2]
```diff
@@ -53,14 +53,22 @@ public static ILogger CreateLogger(IConfiguration configuration, Action<LoggerCo
...
+                                        "/healthz/startup",
+                                        "/healthz/ready",
+                                        "/healthz/live",
+                                        "/metrics",
```
Comment (brandicorel) [inline src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs:63]: Should I add it by default or customize the function to exclude health/metrics routes?

**Review by Gafs63 — APPROVED** [review]

File: src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs [hunk 3]
```diff
@@ -53,17 +53,22 @@ public static ILogger CreateLogger(IConfiguration configuration, Action<LoggerCo
...
 
             configurator?.Invoke(serilogConfig);
 
             Log.Logger      = serilogConfig.CreateLogger();
-            apiHealthLogger = Log.Logger.ForContext(new[] { new PropertyEnricher("SourceContext", "api-health") });
+            apiHealthLogger = Log.Logger.ForContext(new[] {new PropertyEnricher("SourceContext", "api-health")});
             return Log.Logger;
         }
 
+        public static Action<LoggerConfiguration> PathFilterConfigurator(params string[] excludedPaths)
```
Comment (brandicorel) [inline src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs:67]: Is naming correct?

Comment (KevinRecuerda) [reply src/AspNetCore/Configurator/MonitoringConfigurator.Logging.cs:67]: could do extension method ?
in order to use like: `config.Filter.ExcludePaths(...paths)`

**Review by KevinRecuerda — APPROVED** [review]

**Pull request merged**